lib/src/components/NxDrawer/NxDrawer.scss

```diff
@@ -125,6 +125,7 @@
 
   &__close-button {
     grid-area: close-button;
+    height: min-content;
```

Comment:
Feels like it might be cleaner to use `align-content: top` on the header element

Reply:
> Feels like it might be cleaner to use `align-content: top` on the header element

Good call, I've updated it here: 0e555bb70000ba0a3eedea4f432131f169b26c66